config/config.go

```diff
@@ -311,6 +312,9 @@ type (
 
 		// SQLite3 is the config for SQLITE3
 		SQLITE3 SQLITE3 `yaml:"SQLITE3"`
+
+		// InitTryInterval is the config for committee init db
+		InitTryInterval int `yaml:"initTryInterval"`
```

Comment:
Move the config to poll, and try to default to 3sec

Reply:
got it